### ISSUE TITLE
National express

### ISSUE THREAD

**Issue by Identitaet** [osmlab/name-suggestion-index#4404]
National Express = network
https://en.wikipedia.org/wiki/National_Express

Operator for Germany
https://de.wikipedia.org/wiki/National_Express_Holding
I guess internationally https://en.wikipedia.org/wiki/National_Express is the best (unsure tough)

**Comment by Adamant36** [comment]
I almost feel like creating a label just for adding transport entries. It seems odd to use the "add brand" label with them for some reason. I feel like it should be more specific. "add transport brand" maybe? "add brand - transport"? I don't know.

**Comment by Identitaet** [comment]
I'd suggest we go with transit to be consistent but yeah sounds good

**Comment by Adamant36** [comment]
> I'd suggest we go with transit to be consistent but yeah sounds good

Yeah, I hadn't noticed we were going with "transit" until after I wrote the message. Consistent is good though.

**Comment by amous4822** [comment]
National express offers services in Bus, Coach, Train, Tram. Currently it's only added in bus category. Should I add it in other categories too ?

**Comment by bhousel** [comment]
> National express offers services in Bus, Coach, Train, Tram. Currently it's only added in bus category. Should I add it in other categories too ?

Yes!  The script only collects networks that are used on at least 10 routes, so we're surely missing a bunch.

**Comment by KasperFranz** [comment]
It might also be worth changing the wikidata from https://www.wikidata.org/wiki/Q60188 (Mobico Group) to Q13479582  (National Express Coaches)

They renamed their group to Mobico group
source: https://www.expressandstar.com/news/business/2023/05/10/change-of-name-for-national-express/
